addons/sale/static/src/js/sale_order_line_field/sale_order_line_field.xml

```diff
@@ -7,8 +7,8 @@
     >
         <!-- Remove the drag-and-drop handle for combo lines and combo item lines. -->
         <Field position="attributes">
-            <attribute name="t-if">
-                !((isCombo(record) || isComboItem(record)) &amp;&amp; column.widget === 'handle')
+            <attribute name="t-elif">
+                column.widget !== 'handle' || (!props.readonly &amp;&amp; !(isCombo(record) || isComboItem(record)))
```

Comment:
Nit: IMO, it would be a bit more readable as follows :slightly_smiling_face: 
```suggestion
                column.widget !== 'handle' || (!props.readonly &amp;&amp; !isCombo(record) &amp;&amp; !isComboItem(record))
```